internal/command/command.go

```diff
@@ -32,17 +31,14 @@ type Commands struct {
 	zitadelRoles []authz.RoleMapping
 
 	idpConfigSecretCrypto crypto.EncryptionAlgorithm
+	smtpPasswordCrypto    crypto.EncryptionAlgorithm
 
+	UserCodeAlg                 crypto.EncryptionAlgorithm
 	userPasswordAlg             crypto.HashAlgorithm
-	initializeUserCode          crypto.Generator
-	emailVerificationCode       crypto.Generator
-	phoneVerificationCode       crypto.Generator
-	passwordVerificationCode    crypto.Generator
-	passwordlessInitCode        crypto.Generator
 	machineKeyAlg               crypto.EncryptionAlgorithm
 	machineKeySize              int
 	applicationKeySize          int
-	applicationSecretGenerator  crypto.Generator
+	PasswordHashAlg             crypto.HashAlgorithm
```

Comment:
UserCodeAlg and PasswordHashAlg are never used in commands itself, please declare and initialise them where needed